trajectorytools/trajectories.py

```diff
@@ -190,9 +200,7 @@ def acceleration_towards(self, point):
 
     @property
     def distance_to_center(self):
-        print('Warning: this function is deprecated')
-        print('The original center is not remembered nor used')
-        return self.distance_to_origin()
+        raise Exception('Deprecated: Center trajectories and use distance_to_origin')
```

Comment:
A lo mejor indica cual es el método que se tiene que usar para centrar las trayectorias:

```suggestion
        raise Exception('Deprecated: Center trajectories with origin_to and use distance_to_origin')
```